jupyter_client/threaded.py

```diff
@@ -114,13 +107,13 @@ def thread_send():
         assert self.ioloop is not None
         self.ioloop.add_callback(thread_send)
 
-    def _handle_recv(self, future_msg: Awaitable) -> None:
+    def _handle_recv(self, future_msg: asyncio.Future) -> None:
```

Comment:
Looking at [pyzmq's documentation](https://pyzmq.readthedocs.io/en/latest/api/zmq.eventloop.zmqstream.html#zmq.eventloop.zmqstream.ZMQStream.on_recv), shouldn't the argument already be the (multipart) message, not a `Future`?
```suggestion
    def _handle_recv(self, msg_list: List[bytes]) -> None:
```